cmd/security-agent/main_windows.go

```diff
@@ -128,6 +136,41 @@ func (s *service) Run(svcctx context.Context) error {
 				AgentType: catalog,
 			}
 		}),
+		fx.Provide(func() startstop.Stopper {
+			return startstop.NewSerialStopper()
+		}),
+		fx.Provide(func(config config.Component, statsd statsd.Component) (ddgostatsd.ClientInterface, error) {
+			return statsd.CreateForHostPort(pkgconfig.GetBindHost(), config.GetInt("dogstatsd_port"))
```

Comment:
Could we use `setup.GetBindHost(config)`  instead of `pkgconfig.GetBindHost()`  ?